specs/experimental/zk-fault-dipute-game/overview.md

```diff
@@ -13,14 +13,88 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+[g-withdrawal]: ../../glossary.md#withdrawal
+
 ## Overview
 
+The ZK Fault Dispute Game (ZKFDG) is a new type of dispute game compatible with the OP Stack. In ZKFDG, the first
+disagreeing output root is found by the process called "dissection". Then, both derivation of L2 block from L1
+corresponding to that output root and executing the block's transactions is executed in zkVM. The validity of the
+execution is guaranteed by the ZK proof, which can be verified on-chain.
+
+The main difference from [Kroma's previous ZK Fault Proof System](../../fault-proof/challenge.md) is that ZKFDG uses a
+"zkVM" instead of a "zkEVM" to handle all processes. By using zkVM for fault proof, it is enabled to verify all
+processes from derivation to execution without any additional developments of ZK circuit.
+
+ZKFDG implements the OP Stack's `IDisputeGame`, implying that it is fully compatible with the OP Stack's dispute game.
+This also implies that it can be applied to the OP Stack's multi-proof system in the future.
+
 ## ZK Fault Dispute Game Creation
 
+The ZK Fault Dispute Game is created by a validator selected by the
+[`ValidatorManager`](../../protocol/validator-v2/validator-manager.md) at intervals defined by the
+`SUBMISSION_INTERVAL` configured in the [`L2OutputOracle`](../../protocol/validation.md#l2-output-oracle-smart-contract)
+contract. The validator can create a ZKFDG-type dispute game through the `DisputeGameFactory` with an extra data. The
+extra data is composed as follows:
+
+```text
+extra_data = current_l2_block_num ++ prev_game_proxy
+
+current_l2_block_num = uint256
+prev_game_proxy      = bytes32
+```
+
+When the ZKFDG is created by the validator, it is verified that the validator is eligible to submit a claim and the
+`prev_game_proxy` is valid ZKFDG contract. The `prev_game_proxy` is used as the starting output root for the dissection
+process to find the first disagreeing block. If all validations pass, the address of this ZKFDG contract is stored with
```

Comment:
```suggestion
`prev_game_proxy` is the right previous ZKFDG contract. The root claim stored in `prev_game_proxy` is used as the starting output root for the dissection
process to find the first disagreeing block. If all validations are passed, the address of this ZKFDG contract is stored with
```

Reply:
Don't we need the content about current_l2_block_num validation?

Reply:
I'll add it.